scikit_mol/fingerprints.py

```diff
@@ -28,7 +28,15 @@ class FpsTransformer(ABC, BaseEstimator, TransformerMixin):
     def __init__(self, parallel: Union[bool, int] = False, start_method: str = None):
         self.parallel = parallel
         self.start_method = start_method #TODO implement handling of start_method
-    
+
+    @property
+    def _dtype_fingerprint(self):
+        """The dtype of the fingerprint array computed by the transformer
+
+        If needed this property can be overwritten in the child class.
+        """
+        return np.int8
+
```

Comment:
Why not keep this simple and add a hidden property directly to the class?

Reply:
You are right, I used a property to make sure that users were not tempted to override it.
But I agree that the most Pythonic way to achieve this is just to define private attributes.

Changed in https://github.com/EBjerrum/scikit-mol/pull/39/commits/8cb5cfe1c1343a3d1b536695ee52c5c666357cf3